doc/part-1-github-ui/github-contributions.md

```diff
@@ -8,6 +8,16 @@ description: 'PART 1: GITHUB UI'
 
 At this point, the focus is not about Git and Github best practices. The learning videos here will cover the basic things needed to get started with contributing to any project using the Github UI. What you will learn will also help you to tackle the tasks that have been provided for you to do after watching the learning videos.
 
+#### You will learn the following from the learning videos on this page:
```

Comment:
This should not be a sub-heading @oyenuga17
Take a look at the issue again to see that you were not asked to format it.